### DISCUSSION TITLE
Mapbox v10 support available on which rnmapbox release version

### DISCUSSION THREAD

**Discussion by pradeep11** [rnmapbox/maps#1956] [Q&A]
I am using rnmapbox in my react native project now we receive a message from Mapbox that is going to revoke older version support from April 2023. I need to upgrade the mapbox version.

**Comment by mfazekas** [comment]
pls check #main, no released version at the moment

**Reply by ivari** [reply]
Can I ask if there are currently any plans to release some sort of alpha/beta/RC version to NPM in the next, say, month, or should we start looking into integrating v10 from GitHub?

**Reply by mfazekas** [reply]
We've published the first beta to npm:
https://www.npmjs.com/package/@rnmapbox/maps

**Reply by sdrioux** [reply]
Do you have a rough timeline of when you expect to get to a stable release version? Just curious if we can expect it sometime this year or early next?

Thank you!

**Comment by natew** [comment]
I see you are bumping the versions in package.json - why not publish them as well under a tag?

**Reply by mfazekas** [reply]
Good point, added action that bumps version, and creates tag: https://github.com/rnmapbox/maps/tags
